src/web/app/pages-admin/admin-search-page/admin-search-page.component.ts

```diff
@@ -34,6 +35,7 @@ export class AdminSearchPageComponent {
   searchQuery: string = '';
   instructors: InstructorAccountSearchResult[] = [];
   students: StudentAccountSearchResult[] = [];
+  maxResultsToShow: number = ApiConst.SEARCH_QUERY_SIZE_LIMIT_EXCEEDED;
```

Comment:
Does this need to be a class variable? It seems it can be put outside the component, if even need to be a variable (i.e. can use `ApiConst.SEARCH_QUERY_SIZE_LIMIT_EXCEEDED` directly).